lua/starfall/libs_sh/hook.lua

```diff
@@ -383,11 +383,7 @@ add("PlayerFootstep", nil, function(instance, ply, pos, foot, sound, volume)
         sound,
         volume,
     }
-end, function(instance, args, ply)
-    if args[1] and args[2] == true and ply == instance.player then
-        return true
-    end
-end)
+end, returnOnlyOnYourself end)
```

Comment:
that last 'end' is invalid, should delete it

Reply:
Yes fixed it in the next commit after it, my bad